library/std/src/sys/windows/thread_parker.rs

```diff
@@ -113,7 +113,7 @@ impl Parker {
                 // Wait for something to happen, assuming it's still set to PARKED.
                 c::WaitOnAddress(self.ptr(), &PARKED as *const _ as c::LPVOID, 1, c::INFINITE);
                 // Change NOTIFIED=>EMPTY but leave PARKED alone.
-                if self.state.compare_and_swap(NOTIFIED, EMPTY, Acquire) == NOTIFIED {
+                if self.state.compare_exchange(NOTIFIED, EMPTY, Acquire, Acquire) == NOTIFIED {
```

Comment:
This now returns a `Result`, right?

```suggestion
                if self.state.compare_exchange(NOTIFIED, EMPTY, Acquire, Acquire).is_ok() {
```

Reply:
Oh! Of course. Could not run the test locally for Windows. Too long since I did this PR so I forgot it was not all that simple to do the migration :sweat_smile:

Reply:
Thank you @m-ou-se!